src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -378,7 +364,7 @@
           "to be less than this.")                                          \
           range(0, 100)                                                     \
                                                                             \
-  product(uintx, ShenandoahMaxYoungPercentage, 80, EXPERIMENTAL,            \
+  product(uintx, ShenandoahMaxYoungPercentage, 100, EXPERIMENTAL,           \
```

Comment:
If a user sets MinYoung == MaxYoung == 100, does one get single-generation Shenandoah?  Or is that request rejected? What if both are set to 0? Are there sanity checks or tests that would fail today with such settings?